styleguide.md

```diff
@@ -254,3 +254,8 @@ When a sentence starts with inline code,
 this may be rewritten using _The_ inserted before the inline code to avoid a lower case letter at the beginning of the sentence:
 
 > The `import`-clauses are not inherited.
+
+### Use comma after some abbreviations
+
+Always use comma _after_ e.g. and i.e.; note that in some cases a comma is also needed before them.
+This also avoids the need to guard the space after the dot.
```

Comment:
It is a bit more complicated than that. They should be wrapped in punctuation marks. Before them the punctuation marks can be: "(", "--", ",", or "." (end of the previous sentence for "e.g."). After, it is usually "," but ":" is appropriate if what follows starts on a new line (after formatting).

Reply:
Ok, I have tried to add that.

Reply:
Everything else should be implemented, and I found some additional missing commas before.